app/tests/integration/nimbus/parallel_pytest_args.txt

```diff
@@ -4,3 +4,5 @@
 -k KLAR -m run_per_app
 -k FOCUS -m run_per_app
 -k DESKTOP -m run_once
+-k DESKTOP -m run_parallel -n 2
```

Comment:
Fasinating! Per above:

> parallelism: 7 # This should match the number of entries in parallel_pytest_args.txt

Does "-n 2" mean we should increase parallelism by another level? (so 8 instead of 7). Or maybe I'm not fully grasping what `-n` is doing here.

Reply:
this is for pytest, the parallelism thing in the circleci config is for circleci. This adds another circleci parallel job and then tells pytest to run things with 2 workers in parallel.